src/components/language/language.js

```diff
@@ -1,68 +1,41 @@
 import React, { useState, useEffect } from "react";
+import Radio from "../language/radiobuttons";
 import "./language.css";
 
 const Language = props => {
   const [language, setLanguage] = useState();
   const { game } = props;
   useEffect(() => {
     setLanguage(game.language);
-  });
+  }, [game]);
 
   const onChange = e => {
     setLanguage(e.target.value);
     game.setlanguage(e.target.value);
   };
-
+  const languageObj = {
```

Comment:
I would probably give it a more descriptive name like, `languageMap`, `languageDictionary`, etc.